pkg/config/agent/config.go

```diff
@@ -369,21 +370,14 @@ type PolicyBypassRule struct {
 }
 
 type SecondaryNetworkConfig struct {
-	// Secondary network specific OVS configuration.
-	OVS SecondaryNetworkOVSConfig `yaml:"ovs,omitempty"`
-	// TunnelType to be used for node to node transport, which is part of the same virtual network.
-	TunnelType string `yaml:"tunnelType,omitempty"`
+	// Configuration of OVS bridges for secondary networks. At the moment, only a
+	// single OVS bridge is supported.
+	OVSBridges []OVSBridgeConfig `yaml:"ovsBridges,omitempty"`
 }
 
-type SecondaryNetworkOVSConfig struct {
-	// Enable Antrea's native secondary network OVS configuration.
-	Enable bool `yaml:"enable,omitempty"`
-	// OVS integration bridge name.
-	OVSIntegrationBridgeName string `yaml:"ovsIntegrationBridgeName,omitempty"`
-	// OVS transport bridge name.
-	OVSTransportBridgeName string `yaml:"ovsTransportBridgeName,omitempty"`
-	// OVS Datapath type to use for the OpenVSwitch bridge created by Antrea.
-	OVSDatapathType string `yaml:"ovsDatapathType,omitempty"`
-	// OVS patch port which connects the integration and transport bridge.
-	OVSPatchPort string `yaml:"ovsPatchPort,omitempty"`
```

Comment:
Were these ever used anywhere? Are we confident removing the directly wouldn't introduce upgrade issue?

Reply:
No. They were planned to be used for secondary overlay network, but that project has not been moving for > 1 year and I do not see a good chance it will resume any time soon. So, feel good to do a cleanup.